ui/src/logs/actions/index.ts

```diff
@@ -367,7 +368,23 @@ export const executeTableForwardQueryAsync = () => async (
   try {
     dispatch(incrementQueryCount())
 
-    const query = buildForwardLogQuery(time, queryConfig, filters, searchTerm)
+    const upper = await findForwardUpper(
+      time,
+      queryConfig,
+      filters,
+      searchTerm,
+      proxyLink,
+      namespace
+    )
+
+    const query: string = await buildInfiniteLogQuery(
```

Comment:
maybe `buildInfiniteScrollLogQuery`? otherwise it sounds like infinite could map to anything, such as the query, or that i can expect to return an infinite number of logs